designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -327,12 +328,20 @@ class DeploymentServiceImpl(
     } yield result)
   }
 
+  override def getProcessesInProgress: Future[Map[ProcessId, Set[ProcessActionType]]] =
+    dbioRunner.run(
+      actionRepository.getInProgressCanceledAndDeployed
```

Comment:
Let's write a few words why we are interested only about cancel and deploy action types. I mean that we want to fetch as few records as possible and in method ... in DeploymentService we are interested only about these action types.